commands/create.go

```diff
@@ -137,7 +137,8 @@ func runCreate(dockerCli command.Cli, in createOptions, args []string) error {
 		if len(args) > 0 {
 			ep, err = validateEndpoint(dockerCli, args[0])
 			if err != nil {
-				return err
+				logrus.Warnf("No valid Docker endpoint found, will continue with buildkitd endpoint.")
```

Comment:
I'm not sure about this one, but I'm not yet familiar with the argument handling logic, perhaps something to look into.